### PR TITLE
:sparkles: Add finder AppleShowAllFiles command

### PR DESCRIPTION
**Pull request by bpetetot**
Add command `AppleShowAllFiles`
> `Show hidden files in the Finder.`

- Tested on Catalina only
- Added in english and french languages
- Scripts to build screenshots provided (with the new runner)
- Runner modified to avoid capturing window shadow

### CODE REVIEW

**Review by yannbertrand — CHANGES_REQUESTED** [review]
Looks nice! I'll run it on Big Sur to update screenshots but I'm sure the command is working

File: record/finder/AppleShowAllFiles/false.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  run: async (outputPath) => {
+    console.log('> Recording finder show hidden files to false');
+
+    try {
+      const runner = new MacRunner();
+      await runner
+        .setDefault('com.apple.Finder', 'AppleShowAllFiles', '-bool false', 'killall Finder')
+        .openApp('Finder', '~')
+        .captureApp('Finder', `${outputPath}/false.png`)
+        .setDefault('com.apple.Finder', 'AppleShowAllFiles', '-bool true', 'killall Finder')
```
[the remark this answers is not in the document]
Comment (bpetetot) [reply record/finder/AppleShowAllFiles/false.js]: Done 👍

File: record/finder/AppleShowAllFiles/true.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  run: async (outputPath) => {
+    console.log('> Recording finder show hidden files to true');
+
+    try {
+      const runner = new MacRunner();
+      await runner
+        .setDefault('com.apple.Finder', 'AppleShowAllFiles', '-bool true', 'killall Finder')
+        .openApp('Finder', '~')
+        .captureApp('Finder', `${outputPath}/true.png`)
+        .setDefault('com.apple.Finder', 'AppleShowAllFiles', '-bool false', 'killall Finder')
```
[the remark this answers is not in the document]
Comment (bpetetot) [reply record/finder/AppleShowAllFiles/true.js]: Done 👍

**Review by yannbertrand — APPROVED** [review]
LGTM! Thank you

**Pull request merged**